### ISSUE TITLE
[SW/SH] Expert Skirmisher + Eagle Eye/Plink

### ISSUE THREAD

**Issue by anarchypark** [WarEmu/WarBugs#18253]
**Expected behavior and actual behavior:**
Expert Skirmisher tactic reduced Eagle Eye/Plink cast time into 0 making it fire instantly..

**Steps to reproduce the problem:**
spam Eagle Eye/Plink like crazy, several times in 1 second.
first shot is normal. 0.5 cast time.
second shot casting is interrupted while spamming, but shot fire.
after this
all the next shots are instant.

**Testing Screenshots/Videos/Evidences (always needed):**
https://user-images.githubusercontent.com/19988772/103174417-4b982d00-48a5-11eb-9e3d-1320bb9e1301.mp4

I think Festering and Poison also affected by this. seems cast time reduced twice.
couldn't test cuz required CD reducer.
not big game breaking but can be used in pve and lord kill.

**Comment by Kaelang** [comment]
Looks like you're just toggling the visual GCD faster than the server is registering a spell cast, which means theoretically you are casting the skills, but you're lagging behind slightly making it look like it's instant cast. Would need confirmation though before I just close this.

**Comment by anarchypark** [comment]
you're right. both spam and managed clicks shot same amount( 9 ) b4 out of ap.

**Comment by grunbag** [comment]
Yes kaelang is correct
